webrtc.html

```diff
@@ -2438,12 +2438,40 @@ <h4>
                                 <p>
                                   If a suitable transceiver was found
                                   (<var>transceiver</var> is set) and
-                                  <var>sendEncodings</var> is non-empty, set
-                                  <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}}
-                                  to <var>sendEncodings</var>, and set
-                                  <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[LastReturnedParameters]]}}
-                                  to <code>null</code>.
+                                  <var>receiveEncodings</var> is non-empty, run
+                                  the following steps:
                                 </p>
+                                <ol>
+                                  <li class="needs-test">
+                                    <p>
+                                      If
+                                      <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}}
+                                      is non-empty, then for each encoding,
+                                      <var>encoding</var>, in
+                                      <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}},
+                                      copy all of <var>encoding</var>'s members
+                                      and their values to the encoding with the
+                                      same {{RTCRtpCodingParameters/rid}} in
+                                      <var>receiveEncodings</var>, if there is one.
+                                    </p>
+                                    <p>
+                                      Set
+                                      <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}}
+                                      to <var>receiveEncodings</var>, and set
+                                      <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[LastReturnedParameters]]}}
+                                      to <code>null</code>.
+                                      <div class="note">
+                                        If this step produces a net change in an
+                                        existing non-empty
+                                        <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}},
+                                        then the process of applying
+                                        <var>description</var> fails. This
+                                        specification does not allow remotely
+                                        initiated RID renegotiation.
```

Comment:
I don't think you can have a normative failure in a "note". If this is to be a note, it has to point to a "raise error and abort these steps" in a normative section.

Reply:
The note refers to existing normative prose higher up, since it's too late to fail here. Search for "This specification does not allow remotely initiated RID renegotiation." — I'll try to add a link.